### ISSUE TITLE
Dead Rising 2 Off The Record crashes on loading

### ISSUE THREAD

**Issue by NikoBellicRU** [doitsujin/dxvk#1555]
The game opens fine but when i load a savegame or start a new game either it loads but in a bit crashes or imidiatly crashes, also in the main menu sometimes there is no mouse cursor but dont know if thats dxvk or wine.

GPU:GTX 670
Driver:440.48.08
Wine version: Proton-5.0-6-rc2
DXVK version: 1.6

[steam-45770.log](https://github.com/doitsujin/dxvk/files/4440456/steam-45770.log)

**Comment by Joshua-Ashton** [comment]
I'd assume it's possibly the thousands of lines of
``INFO: 0xd693b40 /home/vagrant/proton/FAudio/src/FAudio_internal.c:1764 FAudio_INTERNAL_DecodeWMAERROR 31053ms ERROR: WMA IS NOT SUPPORTED IN THIS BUILD!``

but feel free to try with PROTON_USE_WINED3D=1 to verify that

**Comment by NikoBellicRU** [comment]
Yeah probably, just tried PROTON_USE_WINED3D=1 %command% and still crashed
